### ISSUE TITLE
Timestamp change to date field - mtime

### ISSUE THREAD

**Issue by drewbitt** [simon987/sist2#311]
Feature Request

Timestamp's type should be "date", format should be "epoch_millis". mtime is currently set as type integer which breaks date interpretation integrations in other services like Grafana  

```
{
     "properties": {
         "mtime": {
             "type": "date",
             "format": "epoch_millis"
         }
     }
 }
```

**Comment by simon987** [comment]
Thanks, will be included in the next version

**Comment by simon987** [comment]
Fixed in [c0b8a9c](https://github.com/simon987/sist2/commit/c0b8a9c4673d82ddb6b3380f22ba9594e7495106), will be included in next release 2.13.0
